files/zh-cn/learn/forms/sending_forms_through_javascript/index.md

```diff
@@ -394,35 +348,53 @@ window.addEventListener("load", function () {
     // 建立请求
     XHR.open("POST", "https://example.com/cors.php");
 
-    // 添加需要的 HTTP 头部来处理多重表单数据 POST 请求
+    // 添加需要的 HTTP 头部来处理多部分表单数据 POST 请求
     XHR.setRequestHeader(
       "Content-Type",
-      "multipart/form-data; boundary=" + boundary,
+      `multipart/form-data; boundary=${boundary}`,
     );
 
     // 最后，发送数据。
     XHR.send(data);
   }
 
-  // 访问表单…
-  var form = document.getElementById("myForm");
+  // 获取表单元素
+  const form = document.getElementById("theForm");
 
-  // …接管提交事件
-  form.addEventListener("submit", function (event) {
+  // 添加 submit 事件处理器
+  form.addEventListener("submit", (event) => {
     event.preventDefault();
     sendData();
   });
 });
 ```
 
-在线演示：
+这里是实时演示效果：
 
 {{EmbedLiveSample("处理二进制数据", "100%", 150)}}
 
 ## 总结
 
-取决于不同的浏览器，通过 JavaScript 发送数据可能会很简单，也可能会很困难。{{domxref("XMLHttpRequest/FormData","FormData")}} 对象是通用的答案，所以请毫不犹豫的在旧浏览器上通过 polyfill 使用它：
+取决于不同的浏览器和正在处理数据的类型，通过 JavaScript 发送数据可能会很简单，也可能会很困难。{{domxref("FormData","FormData")}} 对象是通用的答案，所以请毫不犹豫地在旧浏览器上通过 [polyfill](https://github.com/jimmywarting/FormData) 使用它：
+
+## 参见
```

Comment:
BTW, I don't think we really need 'see also' part like this... we've got the sidebars, and the content are just identical 🤔 

![image](https://github.com/mdn/translated-content/assets/40999116/d437ff16-3920-421d-b9aa-cfc0ee0e0803)

Reply:
Similar to #11653. Could you open a discussion for this ;)

Reply:
Discussion opened.

Reply:
I'm going to resolve this because this is out of the PR's scope 👀